dist/components/address/address.js

```diff
@@ -15,6 +15,10 @@ var AddressComponent = (function (_super) {
     AddressComponent.prototype.allowMultiple = function () {
         return false;
     };
+    //return true if component sets an array in setValue...
+    AddressComponent.prototype.isMultipleValueComponent = function () {
+        return true;
```

Comment:
It seems that ```isMultipleValueComponent``` is the inverse of ```allowMultiple```. Are you sure that we cannot just use ```!component.allowMultiple()``` in the if statement check? I would MUCH rather this than introduce what seems to be just the inverse of an existing property.